### PR TITLE
Investigate various portal script additions

### PR DESCRIPTION
**Pull request by maxwelldb**
@kalexand-rh Does this seem to do what you were looking for?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
Yes, this does. Thank you. Please squash so I can merge and confirm it on PV1.

**Comment by maxwelldb** [comment]
@kalexand-rh Done, though might want to avoid merging this for now due to the unresolved includes issue. :cry:

**Comment by kalexand-rh** [comment]
Thank you - I hope we can figure out what's going on with those includes! I will try to do some testing to see if we can work around the errors.

**Comment by maxwelldb** [comment]
@kalexand-rh This should fail in sync at the moment, but I think the master.adoc files it generates will be useful in the future.

File: build_for_portal.py [hunk 1]
```diff
@@ -19,9 +20,9 @@
 import logging
 
 # See manual and pip3 install aura.tar.gz for logging
-from aura import cli
-
-cli.init_logging(False, True)
+# from aura import cli
+#
+# cli.init_logging(False, True)
```
Comment (maxwelldb) [inline build_for_portal.py:25]:
```suggestion
from aura import cli

cli.init_logging(False, True)
```
Uncomment for prod

**Comment by maxwelldb** [comment]
@kalexand-rh Does this still matter at all?

File: build_for_portal.py [hunk 2]
```diff
@@ -1090,6 +1100,38 @@ def parse_repo_config(config_file, distro, version):
...
+def insert_ddf_and_inclusive_modules():
+    """
+    Walk project for all master.adoc files and then insert portal-exclusive DDF and inclusivity modules at the top
+    of those files.
+    """
+
+    replacement = """
+include::includes/making-open-source-more-inclusive.adoc[leveloffset=+1]
+
+include::includes/providing-direct-documentation-feedback.adoc[leveloffset=+1]
```
Comment (maxwelldb) [inline build_for_portal.py:1112]:
```suggestion
include::includes/making-open-source-more-inclusive.adoc[leveloffset=+1]
```

**Pull request closed without merging**